api/v1beta1/events.go

```diff
@@ -273,10 +274,16 @@ var Events = map[EventReason]Event{
 		OnDisruptionTemplateMessage: "DisruptionCron deleted",
 		Category:                    DisruptionCronEvent,
 	},
+	EventDisruptionCleaned: {
+		Type:                        corev1.EventTypeNormal,
+		Reason:                      EventDisruptionCleaned,
+		OnDisruptionTemplateMessage: "Disruption has been cleaned and should be deleted shortly after",
+		Category:                    DisruptionEvent,
+	},
 	EventDisruptionFinished: {
 		Type:                        corev1.EventTypeNormal,
 		Reason:                      EventDisruptionFinished,
-		OnDisruptionTemplateMessage: "DisruptionEvent finished",
+		OnDisruptionTemplateMessage: "Disruption finished",
```

Comment:
👍 oops i missed this one on review last time

Reply:
me too 😄